rpc/tendermint/test/tests.go

```diff
@@ -251,7 +253,59 @@ func asEventDataTx(t *testing.T, eventData txs.EventData) txs.EventDataTx {
 	return eventDataTx
 }
 
-func doNothing(eventId string, eventData txs.EventData) (bool, error) {
+func doNothing(_ string, _ txs.EventData) (bool, error) {
 	// And ask waitForEvent to stop waiting
 	return true, nil
 }
+
+func testSubscribe(t *testing.T) {
```

Comment:
Oh actually since I factored out `readResult`. I could probably get away with having it in client_ws_test.go. Still I'm fairly happy having it here since all the tests in client_ws_test.go are proxy functions to ones here. Mind you they need to be to centralise the HTTP/JSON logic.